src/resonanceReconstruction/breitWigner/multiLevel/Apply.hpp

```diff
@@ -3,5 +3,8 @@ class Apply : protected singleLevel::Apply {
   #include "resonanceReconstruction/breitWigner/multiLevel/Apply/src/build.hpp"
 
 public:
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wreturn-type"
   #include "resonanceReconstruction/breitWigner/multiLevel/Apply/src/call.hpp"
```

Comment:
The call operator is entirely inside a `try` block. If no exception is thrown, there is always a `return` statement. Otherwise, an exception is thrown.

There may be a better way of doing this, but I don't know what it is.

Reply:
That's an annoying one. I wonder why the unit test that includes this does not complain about it.

Reply:
Even adding a return statement won't solve it since it would be unreachable code.